ansible/roles/ocp4-workload-pam-fraudmanagement-workshop/tasks/workload_per_project_businessautomationworkload.yml

```diff
@@ -166,10 +166,15 @@
 
 - name: Maven install for project {{ _case_kie_project_id }}
   uri:
-    url: "https://{{ businesscentral_host.stdout }}/rest/spaces/workshop/projects/{{ _case_kie_project_id }}/maven/install"
+    url: >-
```

Comment:
Please try to clean this up per previous comments. You can format it or you can increase the line length and keep it on a single line.